### PR TITLE
Feature/view recipe info

### PR DESCRIPTION
**Pull request by fernandorobles97**
What I did:
- Deleted unnecessary console.logs.
- Added singleRecipeDisplay DOM element; added event listener to display specific recipe that is selected.
- Add viewSelectedRecipe function to allow the user the ability to see all information regarding that recipe.
- Refactored calculateRecipeCost by adding an additional parameter to take in a specific data model.
- Passed additional argument for calculateRecipeCost invocation in viewSelectedRecipe function.
- Fixed import of ingredientsData to come from the correct source.
- Refactored findIngredientNames function to use .join so there's a space added between words.
- Added singleRecipeDisplay to addHiddenClass invocation in showRecipes function.

Did this break anything?
- [x] No

Type of change
- [x] New feature (non-breaking change which adds functionality)
- [x]  Bug fix (non-breaking change which fixes an issue)
- [x]  Refactor(DRY-ing up/ reorganizing code, etc.)
- [x]  Super small fix (Corrected a typo, removed a comment, etc.)

Checklist:
- [x]  If this code needs to be tested, all tests are passing
- [x]  The code runs locally
- [x]  There are comments where clarification/ organization is needed
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing

What's next?
Moving onto iteration 2 and creating the user data functions.

### CODE REVIEW

**Review by hvohr — APPROVED** [review]
Love the way you constructed the calculateRecipeCost function, very DRY!

File: src/domUpdates.js [hunk 1]
```diff
@@ -24,24 +26,28 @@ searchInput.addEventListener('keypress', function (e) {
...
+  viewSelectedRecipe(event);
+});
+
 //Event Handlers/Functions
 function showSearchResults() {
   let searchValue = searchInput.value
   removeHiddenClass([allRecipeDisplay, allFilterDisplay])
   addHiddenClass([frontRecipeDisplay]);
     allRecipeDisplay.innerHTML = ''
-    console.log(recipesfromName(recipeData, searchValue).forEach(recipe => allRecipeDisplay.innerHTML += `<div class = "recipe-wrapper">
```
Comment (Denzel-Turambi) [inline src/domUpdates.js:33]: Good catch on the console.log!

File: src/domUpdates.js [hunk 2]
```diff
@@ -56,15 +62,33 @@ function renderFilteredRecipes() {
...
+  singleRecipeDisplay.innerHTML= `
+  <h2>${selectedRecipe.name}</h2>
+  <img id="${selectedRecipe.id}" src="${selectedRecipe.image}" class="recipe" alt='${selectedRecipe.name}'>
+  <p class="total-cost-box">This recipe costs a total of: $${recipeCost} to make!</p>
+  <p class="ingredient-box">The ingredients you will need to make this recipe are: <br>
+  ${ingredients}</p>
+  <p class="instruction-box">Instructions: <br>
+  ${instructions}</p>`;
+}
+
```
Comment (Denzel-Turambi) [inline src/domUpdates.js:81]: I love how this function is formatted! I like how you assigned variables to each invocation.

**Review by Denzel-Turambi — APPROVED** [review]

**Review by andreasorensen — APPROVED** [review]
Thanks for removing that error handling! I'll delete those tests ASAP. It def doesn't make sense to have them, since these functions will only be invoked with a click event. I wasn't sure at first, but happy to get that done.

**Pull request merged**